src/applications/vaos/containers/AppointmentsPage.jsx

```diff
@@ -79,18 +92,43 @@ export class AppointmentsPage extends Component {
                 startNewAppointmentFlow={this.startNewAppointmentFlow}
               />
             )}
-            {hasExpressCareAccess && (
+            {!showExpressCare && (
               <>
-                <RequestExpressCare />
-                <h2 className="vads-u-font-size--h3 vads-u-margin-y--3">
-                  View your upcoming, past, and Express Care appointments
-                </h2>
+                {showPastAppointments && <TabNav />}
+                {children}
               </>
             )}
-            {showPastAppointments && (
-              <TabNav hasExpressCare={hasExpressCareAccess} />
+            {showExpressCare && (
+              <>
+                {(futureStatus === FETCH_STATUS.loading ||
+                  futureStatus === FETCH_STATUS.notStarted) && (
+                  <LoadingIndicator message="Loading your appointment information" />
+                )}
+                {futureStatus === FETCH_STATUS.succeeded && (
+                  <>
+                    <RequestExpressCare />
+                    {hasExpressCareRequests && (
+                      <h2 className="vads-u-font-size--h3 vads-u-margin-y--3">
+                        View your upcoming, past, and Express Care appointments
+                      </h2>
+                    )}
+                    <TabNav showExpressCare={hasExpressCareRequests} />
+                    {children}
+                  </>
+                )}
+                {futureStatus === FETCH_STATUS.failed && (
```

Comment:
We could change this to not show the error on the past appointments page, so that you can get to to your past appointments if requests are failing.

Reply:
Wouldn't this show 2 failed messages on the future list? It might make more sense to keep the failure alertBox logic in each list and display based on the statuses, but maybe I'm missing something

Reply:
Yeah, I think you're right that I would need to also condition the alert in FutureAppointmentList on the express care flag.

I moved the error alert up because we can't reliably determine what tabs to show if there's an error pulling requests. But you might be right that it'll be a little easier to understand if I move it back down and update the tab nav to show the EC tab when you're already on that page and there's an error.

Reply:
Moved the alerts back into the tabs